custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -380,15 +382,24 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
             'format': vm.selectedFormat,
             'pdfformat': vm.selectedPDFFormat,
             'selected_awcs': awcs.join(','),
-        }).then(function(data) {
+        };
+        var selectedFilters = vm.selectedFilterOptions();
+        if (vm.isChildBeneficiaryListSelected()) {
```

Comment:
why does this need to act differently than other reports?

Reply:
As this report only gets additionally a list of filters.